### PR TITLE
Try to turn audit mode back on without building test/utilities

### LINKED ISSUE #2098: Restore static analysis builds when we can get the build machines to have capacity
@DHowett-MSFT has active threads with the build people to figure out the capacity issue. I had to turn off static analysis builds in #2096 temporarily because it is out of disk space and we need to keep moving. When the capacity issue is figured out, this represents uncommenting and turning them back on.

### PR DESCRIPTION
**Pull request by miniksa**
## Summary of the Pull Request
Audit mode was turned off because it ran out of disk. We talked to the Azure Pipelines Managed Agents team. They gave us some ideas. Then I had one too: don't audit test libraries and utilities and maybe we'll save enough space to get through this.

## PR Checklist
* [x] Closes #2098
* [x] I'm an employee
* [x] This is a test
* [x] Doesn't need documentation
* [x] I am a core contributor

## Validation Steps Performed

This happens on build agents so this PR *is* the validation.

### CODE REVIEW

**Comment by miniksa** [comment]
Hey look, @adiviness, I fixed it. :D

I checked this locally and we drop 2-4GB by not building the test things in audit mode. Should be good enough to get us back online.

File: OpenConsole.sln [hunk 1]
```diff
@@ -466,7 +466,6 @@ Global
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|ARM64.ActiveCfg = Release|ARM64
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|ARM64.Build.0 = Release|ARM64
 		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|x64.ActiveCfg = Release|x64
-		{531C23E7-4B76-4C08-8AAD-04164CB628C9}.AuditMode|x64.Build.0 = Release|x64
```
Comment (zadjii-msft) [inline OpenConsole.sln]: Should we remove the rest of the AuditMode stuff from these projects, or just leave them with only the `AuditMode|x64.Build.0` removed?

Comment (miniksa) [reply OpenConsole.sln]: They don't actually have an audit mode configuration defined in the projects. It's just that the solution has one and it has to map to SOMETHING in the individual projects.

Comment (miniksa) [reply OpenConsole.sln]: OK. we talked and you meant the x86 and ARM ones too. I dropped those too.

**Comment by miniksa** [comment]
Also, this reverts #2096 effectively.

**Comment by adiviness** [comment]
I’m not thrilled about the tests not being audited as well, but it’s better than nothing. Is there an issue tracking the re-enabling of test auditing?

**Comment by miniksa** [comment]
> I’m not thrilled about the tests not being audited as well, but it’s better than nothing. Is there an issue tracking the re-enabling of test auditing?

No. I mean, they weren't actually being audited to begin with. They were still building release mode anyway. I presume that whenever we get around to expanding the audit through all the production code that we might consider the test and utility code after that. But we've always been faster/looser with test/utility code than production code.

**Pull request merged**